lib/components/viewers/route-viewer.js

```diff
@@ -1,56 +1,88 @@
 import coreUtils from '@opentripplanner/core-utils'
-import React, { Component, PureComponent } from 'react'
-import PropTypes from 'prop-types'
-import { Label, Button } from 'react-bootstrap'
-import { VelocityTransitionGroup } from 'velocity-react'
+import React, { Component } from 'react'
 import { connect } from 'react-redux'
-import styled from 'styled-components'
+import PropTypes from 'prop-types'
```

Comment:
Sort imports. (I recall we enabled prettier automatically on commits at one point?)

Reply:
Not in this repository yet I don't think. https://github.com/opentripplanner/otp-react-redux/pull/447 adds a new eslint config that includes automatic sorting.

Until that's merged, manually sorted in 9a7139548d78db23db07dfddf9940bd85078072b